README.md

```diff
@@ -1,24 +1,169 @@
+
+- Follow our list of [best practices for Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<ul>
+  <li>Ruby</li>
+  <li>Ruby on Rails</li>
+  <li>JavaScript</li>
+  <li>PostgreSQL</li>
+  <li>GitHub</li>
+</ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Project 1: Creating a data model.
+- Project 2: Validations and Model specs.
+- Project 3: Processing data in models.
+- Project 4: Setup and controllers.
+- Project 5: Controllers specs.
+- Project 6: Views.
+- Project 7: Forms.
+- Project 8: Integration specs for Views and fixing n+1 problems.
+- Project 9: Add Devise.
+- Project 10: Add authorization rules.
+- Project 11: Add API endpoints.
+- Exercise: API documentation.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo & Project Presentation <a name="live-demo"></a>
+
+- Live Demo coming soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+### Setup
+- Clone this repository to your computer: git clone https://github.com/ONJoseph/BlogApp.git
+- Open the project on your code editor: "code BlogApp"
+
+### Install
+- Write: bundle install
+- Write: npm install
+
+### Usage
+- To test the app:
+    - rspec (general)
+    - rspec spec/unit/name_of_sile (specific one)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Joseph Ogbole**
+
+- GitHub: [@ONJoseph](https://github.com/ONJoseph)
+- Twitter: [@ONJoseph1](https://twitter.com/ONJoseph1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/o-n-joseph-ba8425147/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add more interactivity to the function.
+- [ ] Add more methods to test BlogApp development.
+- [ ] Add more unique features.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ONJoseph/BlogApp/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+I really enjoyed working on this project. If you like it show your support by giving a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Acknowledgments -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Hat tip to anyone whose code was used.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As the previous reviewer suggested, Please include an MIT license file in your project. The link to your license should be included in your README file. The MIT License is a popular open-source license that allows others to use, modify, and distribute your code as long as they give credit to you the original author, and include the original license text. You can use this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project. :+1: